File: portfolio/java/새 폴더/중간고사01/src/중간고사01/sum.java
package 중간고사01;
import java.util.*;
public class sum {
	public static void main(String[] args) {
		Scanner key = new Scanner(System.in);
		int max=1;
    do{
    	System.out.print("정수 입력");
    	int z=key.nextInt();
    	if(max<z){
    		max=z;
    	}
    	if(z<=0){
    		break;
    	}
    }while(true);
    System.out.print(max);  
	}
}

File: portfolio/c_c++/c++/명품CPP프로그래밍(개정판)_학습자용/연습문제 실습 정답이 공개된 문제들/prac13/13-1/sum.cpp
#include <iostream>
using namespace std;

int sum(int a, int b) {
	if(a > b) throw "잘못된 입력";
	if(a < 0 || b < 0) throw "음수 처리 안됨";
	int s = 0;
	for(int i=a; i<=b; i++)
		s += i;
	return s;
}

int main() {
	try {
		cout << sum(2,5) << endl;
		cout << sum(-1,5) << endl;
	}
	catch(const char* s) {
		cout << s << endl;
	}
}